rules/framework.mk

```diff
@@ -0,0 +1,19 @@
+# framework package
+
+FRAMEWORK = framework_1.0.0_$(CONFIGURED_ARCH).deb
```

Comment:
>framework [](http://example.com/codeflow?start=12&length=9)

framework is a confusing name, according to HLD, you want to name it as 'WarmbootManager' ?

Reply:
Shared the alternative names in the other thread, can we go with sysmgr ?